sirepo/template/warpvnd.py

```diff
@@ -689,11 +695,10 @@ def _extract_optimization_results(run_dir, data, args):
 
 def _extract_particle(run_dir, model_name, data, args):
     limit = int(args.renderCount)
-    hf = h5py.File(str(run_dir.join(_PARTICLE_FILE)), "r")
-    d = template_common.h5_to_dict(hf, "particle")
-    kept_electrons = d["kept"]
-    lost_electrons = d["lost"]
-    hf.close()
+    with open(_PARTICLE_FILE, "rb") as f:
```

Comment:
I think in general we should convert files for existing simulations when we change formats, if possible, though in practical terms it may not matter here

Reply:
I think in this case, just getting the app working again is good enough.